nightwatch.js

```diff
@@ -1,6 +1,10 @@
 'use strict'
-
-var TRAVIS_JOB_NUMBER = process.env.TRAVIS_JOB_NUMBER
+var TEST
+if (process.env.TRAVIS_JOB_NUMBER === 'test' || process.env.TRAVIS_JOB_NUMBER === undefined) {
```

Comment:
yes. the `.env` file contains an entry `TUNNEL_IDENTIFIER` which can and should or currently has to be overriden in `.env.local`.
I set it to `TUNNEL_IDENTIFIER=test` in my `.env.local`